src/main/scala/mailo/package.scala

```diff
@@ -87,6 +91,7 @@ class Mailo(
         attachments = attachments,
         tags = tags
       ))
+      _ = logger.info(s"email sent with result: $result")
```

Comment:
Better to stick with `.debug`, imho.

Reply:
I deliberately decided to stick with `info`. This is unlikely going to fill the disk with annoying info (if that is the case, it means we are spamming too many emails :p).
I think it's a useful information, if you really want you can anyhow filter it out later with log4j or logback.

The idea was to allow the user to log one line per email sent. Debug level currently contains too much stuff.